src/TheThingsNetwork.h

```diff
@@ -44,21 +45,18 @@ class TheThingsNetwork
     bool enableFsbChannels(int fsb);
 
   public:
-    TheThingsNetwork(fp_ttn_t fp)
-    {
-      this->fp = fp;
-    }
     int downlinkPort;
     byte downlink[64];
+    TheThingsNetwork(fp_ttn_t fp);
     void init(Stream& modemStream, Stream& debugStream);
     void reset(bool adr = true, int sf = DEFAULT_SF, int fsb = DEFAULT_FSB);
     bool personalize(const byte devAddr[4], const byte nwkSKey[16], const byte appSKey[16]);
     bool join(const byte appEui[8], const byte appKey[16]);
     int sendBytes(const byte* buffer, int length, int port = 1, bool confirm = false);
     int sendString(String message, int port = 1, bool confirm = false);
     void showStatus();
-    void configure_channels(int total_ch);
-    void set_dcycle();
+    void configure_channels();
+    void configure_EU868();
```

Comment:
This is not public, that's the point of passing a frequency plan, we hide it from the user